### ISSUE TITLE
Team > People bewerken in CMS

### ISSUE THREAD

**Issue by Marieke-DIVD** [DIVD-NL/web-www-v2#375]
Op dit moment kan ik geen people bewerken in het CMS. Als het goed is is het de bedoeling dat  het CMS de input vanuit TheOrg kan overrulen. 

![Image](https://github.com/DIVD-NL/web-www-v2/assets/148967054/6cc676b0-f681-4e56-bac3-5f0388e1d4ca)

**Comment by MagicLegend** [comment]
Dat is correct. Dat kan wel, maar je moet de persoon inderdaad met de hand aanmaken :) Matching gebeurd op basis van de slug, die moet 1:1 kloppen met wat er in TheOrg staat.

**Comment by Marieke-DIVD** [comment]
Dus als er nu al iemand zichtbaar is als people op de website, en ik wil iets wijzigen, dan moet ik eerst een people aanmaken in het CMS. Vervolgens alles copy+pasten vanuit TheOrg en dan aanpassen wat ik aangepast wil hebben?

**Comment by MagicLegend** [comment]
Nee, het script houdt ook alleen rekening met de image en description velden (hier hebben we toen ook uitgebreid over gesproken, en alleen deze twee velden wou je een override voor hebben). De rest wordt in alle gevallen overschreven met wat er in TheOrg staat. Wat je niet invult wordt automatisch aangevuld met wat erin TheOrg staat.

Voor future reference, dat gebeurd hier: https://github.com/DIVD-NL/web-www-v2/blob/5e737c4efe147cc14f235c3208f6ae90f95f075c/scripts/generate_people.js#L96-L98

**Comment by Marieke-DIVD** [comment]
This issue might not be relevent after re-structure the teampages.
